dashboard/app/models/authentication_option.rb

```diff
@@ -62,6 +62,10 @@ class AuthenticationOption < ApplicationRecord
     MICROSOFT
   ]
 
+  def google_oauth2?
```

Comment:
I'd call this either 
```suggestion
  def google_oauth?
```

or just 
```suggestion
  def google?
```